INSTALL.rst

```diff
@@ -87,65 +89,80 @@ For example, on Debian GNU/Linux, you can install them as follows:
                         rabbitmq-server \
                         redis-server
 
-and do a system install for the Sass preprocessor by following `Sass web guide <http://sass-lang.com/install>`_ and running:
+Now, add the following lines in your "elasticsearch.yml" (for
+Debian GNU/Linux the full path is
+``/etc/elasticsearch/elasticsearch.yml``):
 
 .. code-block:: shell
 
-  sudo npm install -g sass node-sass clean-css uglify-js requirejs
-
+  # MY CONFIGS
 
-Installation
-^^^^^^^^^^^^
+  cluster.name: cap
+  discovery.zen.ping.multicast.enabled: false
+  http.port: 9200
+  http.publish_port: 9200
 
-Let's start by creating a new virtual environment that will hold our CAP
-v3.0 instance:
+Finally, do a system install (see below for how to do a local install
+instead) for the Sass preprocessor by following
+`Sass web guide <http://sass-lang.com/install>`_ and running:
 
 .. code-block:: shell
 
-   mkvirtualenv cap
+  sudo npm install -g node-sass clean-css uglify-js requirejs
 
-Start redis server in the background:
+Installation
+^^^^^^^^^^^^
+
+Let's start by cloning the repository:
 
 .. code-block:: shell
 
-   redis-server &
+   git clone https://github.com/cernanalysispreservation/analysis-preservation.cern.ch.git cap
+
+Environment Setup
+"""""""""""""""""
 
-Install CAP package:
+All else will be installed inside a python *virtualenv* for easy
+maintenance and encapsulation of the libraries required. To do so,
+create a new virtual environment to hold our CAP instance
+from inside the repository folder:
 
 .. code-block:: shell
 
-   cdvirtualenv
-   mkdir src && cd src
-   git clone https://github.com/cernanalysispreservation/analysis-preservation.cern.ch.git cap
    cd cap
-   pip install -r requirements.txt
+   mkvirtualenv cap_venv
 
-Add the following lines in your "elasticsearch.yml":
+Start redis server in the background:
 
 .. code-block:: shell
 
-  # MY CONFIGS
-
-  cluster.name: cap
-  discovery.zen.ping.multicast.enabled: false
-  http.port: 9200
-  http.publish_port: 9200
+   redis-server &
 
-Run npm to install any necessary JavaScript assets the Invenio modules
+Install the CAP package from inside your ``cap`` repository folder and
+run npm to install the necessary JavaScript assets the Invenio modules
 depend on:
 
 .. code-block:: shell
 
-   cd cap
+   pip install -r requirements.txt
    cap npm
    cdvirtualenv var/cap-instance/static
    npm install bower
    npm install
+
+Install alpaca:
+
+.. code-block:: shell
+
    cd node_modules/alpaca
    npm install
    npm start
```

Comment:
I don't have `bower_components` and I have `build` instead of `dist` (also, `alpaca` is `command not found`). :confused: 
Plus, Javier changed the settings from `dist` to `build`, should we change them back again? I can try that.